db/seeds.rb

```diff
@@ -833,3 +833,20 @@ def seed_quantity(item_name, organization, storage_location, quantity)
   ]
 )
 TransferCreateService.call(transfer)
+
+# ----------------------------------------------------------------------------
+# Users invitation status
+# ----------------------------------------------------------------------------
+# Mark users `invitation_status` as `accepted`
+# 
+# Addresses and resolves issue #4689, which can be found in:
+# https://github.com/rubyforgood/human-essentials/issues/4689
+def mark_invitation_status_as_accepted!(user)
+  user.invitation_sent_at = Time.current
+  user.invitation_accepted_at = Time.current
+  user.save!
+end
+
+User.where(invitation_token: nil).each do |user| 
+  mark_invitation_status_as_accepted!(user)
```

Comment:
```suggestion
User.where(invitation_token: nil).each do |user| 
  user.update!(invitation_sent_at: Time.current, invitation_accepted_at: Time.current)
end
```

Can do this a lot more succinctly. :grin:

Reply:
Totally agreed! 🙂